nix/pyenv.nix

```diff
@@ -33,15 +33,18 @@ let
         pydantic # >=1.7,<1.9 TODO: check if this version is correct/if it matters
       ];
     };
-in pkgs.python310.withPackages (ps: [
-  ps.pep8
-  ps.pytest # ==5.3.5 TODO: check if this version is correct/if it matters
-  ps.pytest-xdist # ==1.34.0 TODO: check if this version is correct/if it matters
-  sslyze # ==5.0.2 TODO: check if this version is correct/if it matters
-  ps.pytest-rerunfailures
-  ps.tox
-  ps.typing-extensions
-  ps.setuptools-rust
-  ps.cryptography
-  nassl
-])
+in pkgs.python310.withPackages (ps:
+  [
+    ps.pep8
+    ps.pytest
+    ps.pytest-xdist
+    ps.pytest-rerunfailures
+    ps.typing-extensions
+    ps.setuptools-rust
+    ps.cryptography
+  ] ++ (if pkgs.stdenv.hostPlatform.system == "x86_64-linux" then [
```

Comment:
The sslyze test seems to run for every platform except aarch64, but the sslyze dependency is installed only on linux x86_64. Is there any concern about running the integ tests on a platform other than aarch64 or linux x86-64? Like, if you were to try and run the integ tests on mac, would it attempt to run sslyze but fail because the dependency isn't installed?

Reply:
Just being cautious here, the other possible platform that might get these packages is `x86_64-darwin`, which we'll add when doing the darwin nix work. If the dependency can't be installed, the devShell won't even start.